docs/core/windows-prerequisites.md

```diff
@@ -11,7 +11,7 @@ ms.devlang: dotnet
 ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 ---
 
-# Prerequisites for Windows development
+# Prerequisites for Windows development and/or running .Net Core applications
 
 .NET Core development on Windows with Visual Studio requires:
```

Comment:
I'd probably change a few sections around. For example, I'd create a section for Visual Studio after the general information about .NET Core on Windows and move this info further down. But I can make the changes for you after we merge this.